### PR TITLE
[JSON] getBVal -> getBool; getFNum -> getFloat; getNum -> getBiggestInt; add getInt

### PR DESCRIPTION
**Pull request by Yardanico**
Changed procedure names, so they make more sense now.
Old procedure names are now deprecated.

### CODE REVIEW

File: tests/niminaction/Chapter9/configurator/configurator.nim [hunk 1]
```diff
@@ -47,7 +47,7 @@ proc createLoadProc(typeName: NimIdent, identDefs: seq[NimNode]): NimNode =
         `cfgIdent`.`fieldNameIdent` = `objIdent`[`fieldName`].getStr
     of "int":
       body.add quote do:
-        `cfgIdent`.`fieldNameIdent` = `objIdent`[`fieldName`].getNum().int
```
Comment (dom96) [inline tests/niminaction/Chapter9/configurator/configurator.nim]: Please don't change this. We need to make sure that whatever is in the book continues to work. That's the whole point.

File: web/bountysource.nim [hunk 2]
```diff
@@ -53,7 +53,7 @@ proc processSupporters(supporters: JsonNode) =
   echo("Found ", supporters.elems.len, " named sponsors.")
```
Comment (dom96) [inline web/bountysource.nim]: Just remove this file please, it lives in the website repo now.

**Comment by konqoro** [comment]
So how about the fields of JsonNode?? Their named str, num, fnum, bval and are exported. I would ask to revert this.

**Comment by Yardanico** [comment]
@konqoro generally fields of JsonNode shouldn't be used directly.

**Comment by konqoro** [comment]
ok @Yardanico how about properly deprecating these procs without removing them? This change breaks a lot of code out there for no reason.

**Comment by Yardanico** [comment]
@konqoro check the code please, I've deprecated old proc names

**Comment by konqoro** [comment]
@Yardanico no you didn't code does not compile

**Comment by Yardanico** [comment]
@konqoro are you sure?

**Pull request merged**